packages/host/app/components/ai-assistant/code-block/command-header.gts

```diff
@@ -61,6 +61,7 @@ const CodeBlockCommandHeader: TemplateOnlyComponent<CodeBlockCommandHeaderSignat
         letter-spacing: var(--boxel-lsp-xs);
         line-height: 1.5em;
         text-wrap: pretty;
+        overflow-wrap: break-word;
```

Comment:
Are we ok with the word potentially breaking into something like this:
```
Canc
el
```

Reply:
I don't love that, but I am OK with it if that is the only way to avoid breaking the layout. WDYT?

Reply:
Alternative is using ellipsis. Up to you